src/boot_fake_node/mod.rs

```diff
@@ -327,12 +327,27 @@ async fn fetch_latest_release_tag_or_local(owner: &str, repo: &str) -> Result<St
     }
 }
 
+pub async fn fetch_kinostate() -> Result<()> {
+    let json_path = format!("{}/kinostate.json", KIT_CACHE);
+    let json_url = "https://gist.githubusercontent.com/bitful-pannul/be02f6085dbc4bc8a4cda0c40039fb3c/raw/9b51cc3116b6a3839c3c42e8aa6bcacb43f4753d/kinostate.json";
+
+    // TODO: statefile versioning
```

Comment:
I think I'll try to use env::var("CARGO_MANIFEST_DIR")?) or similar to fetch it from the installed kit binary if possible instead of from remote http. 

The versioning problem is essentially if we upgrade KNS contracts, runtime reflects it but this kit won't

Reply:
Another option is to `include!()` it if you just want to build it into the binary.